docs/running-on-kubernetes.md

```diff
@@ -990,6 +990,52 @@ See the [configuration page](configuration.html) for information on Spark config
   Specify whether executor pods should be deleted in case of failure or normal termination.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.kubernetesClient.submission.connectionTimeout</code></td>
+  <td>10000</td>
+  <td>
+    Connection timeout in milliseconds for the kubernetes client to use for starting the driver. In client mode, use
+    <code>spark.kubernetes.kubernetesClient.connectionTimeout</code> instead.
```

Comment:
For the variable name prefix, how about changing from `spark.kubernetes.kubernetesClient` to `spark.kubernetes.client` simply?